sdk/src/main/java/com/mapbox/maps/renderer/widget/BitmapWidget.kt

```diff
@@ -0,0 +1,29 @@
+package com.mapbox.maps.renderer.widget
+
+import android.graphics.Bitmap
+
+open class BitmapWidget(
+  bitmap: Bitmap,
+  position: WidgetPosition = WidgetPosition.BOTTOM_LEFT,
+  marginX: Float = 0f,
+  marginY: Float = 0f,
+) : Widget() {
+  override val renderer = BitmapWidgetRendererImpl(
+    bitmap = bitmap,
+    position = position,
+    marginX = marginX,
+    marginY = marginY,
+  )
+
+  override fun updateBitmap(bitmap: Bitmap) {
+    renderer.updateBitmap(bitmap)
+  }
+
+  override fun translate(translateX: Float, translateY: Float) {
+    renderer.translate(translateX = translateX, translateY = translateY)
+  }
+
+  override fun rotate(angleDegrees: Float) {
```

Comment:
perhaps `rotateBy`? or is it final angle and not delta?... again, missing docs here and above

Reply:
Yes it sets absolute (not relative to current) rotation
Adding docs

Reply:
Added docs, renamed to `setTranslation` / `setRotation`